htdocs/product/class/product.class.php

```diff
@@ -5230,7 +5230,7 @@ public function getLabelOfUnit($type = 'long')
         $resql = $this->db->query($sql);
         if ($resql && $this->db->num_rows($resql) > 0) {
             $res = $this->db->fetch_array($resql);
-            $label = ($label_type == 'short' ? $res[$label_type] : 'unit'.$res['code']);
+            $label = ($label_type == 'short' ? $res[$label_type] : $res['code']);
```

Comment:
The translation of units are saved into language file with syntax 'unitXXX' with XXX is unit code.
If you removed this, the units will not more be translated.

Reply:
Did not aware it is translation, revert and added the translation, its work like charm. thanks for the guidance.